week-02/seonju-docker.md

```diff
@@ -0,0 +1,98 @@
+# Docker
+
+## What is Container?
+
+Before we look into Docker, let's suppose when we publish a web application.
+
+- buy a server
+- install Linux
+- set up the software technology to build websites and web applications
+    - the operating system, the web server, the database server and the programming language
+- run the app
+
+In additional, if your app got popular, you practiced good load balancing by setting up a second server to ensure the application wouldn't crash from too much traffic.
```

Comment:
I noticed a small grammar mistake in your writing. You wrote 'in additional', but the correct phrase is 'in addition'. Just thought I'd mention it in case it helps for future writing.